### ISSUE TITLE
Email Comment: Joint Comments from the Council on Governmental Relations and the Association of American Universities

### ISSUE THREAD

**Issue by OMBPublicComments** [WhiteHouse/source-code-policy#148]
[Email Comment, received 4/11/2016 at 12:51PM ET]

[Open Source Policy Comments-COGR_AAU.pdf](https://github.com/WhiteHouse/source-code-policy/files/213603/Open.Source.Policy.Comments-COGR_AAU.pdf)

www.aau.edu • 1200 New York Ave., NW, Suite 550, Washington, DC 20005 • (202) 408-7500
www.cogr.edu • 1200 New York Ave., NW, Suite 460, Washington, DC 20005 • (202) 289-6655

April 11, 2016
OMB Office of the
Chief Information Officer
Submitted via sourcecode@omb.eop.gov

Re: OMB’s Draft Open Source Policy

Dear Sir or Madam,

The Council on Governmental Relations (“COGR”) is an association of 190 U.S. research universities and their affiliated academic medical centers and research institutes that concerns itself with the impact of federal regulations, policies, and practices on the performance of research and other sponsored activities conducted at its member institutions. The Association of American Universities (“AAU”) is a nonprofit association of 60 U.S. and two Canadian public and private research universities. Founded to advance the international standing of U.S. research universities, AAU today focuses on issues that are important to research-intensive universities, such as funding for research, research regulations, and graduate and undergraduate education.

COGR, AAU and our member universities support efforts to provide public access to and preservation of the results of research funded by Federal agencies. The United States Government’s investment in university research has produced tremendous benefits for the nation’s economic competitiveness and national security. Ensuring appropriate access to research results not only benefits the public at large, but also accelerates further research of the nation’s scientists and scholars.

We appreciate the intent of the proposed policy to improve access to custom software developed for the Federal Government and, in particular, the reuse of custom source code, associated documentation, and related files across the various agencies of the Federal Government. And we support the proposal to the extent that it will increase utilization and reduce administrative burdens without unduly compromising the commercialization of research results.

Unfortunately, there are portions of the proposed policy that we believe would significantly increase administrative burdens, ambiguity, and result in a loss of commercialization opportunities, including new company formation, that lead to further innovation. These include: the definition of “custom code” to include not only the software solution itself, but also the customization of existing software to meet the Federal Government’s need and the creation of stand-alone modules; the requirement to secure unlimited rights to the custom source code without consideration of Federal Acquisition Regulations (FARS) Section 252.227-7014 Rights in Noncommercial Software Alternative I for Institutions of Higher Education and other small businesses; the establishment of a twenty percent (20%) minimum to be released as OSS without taking into account the type of custom code being developed or the possible impact on commercialization; and the lack of clarity on what open source license language will govern the release of a particular custom code and what rights, if any, the copyright owner may have in the release of custom code. Furthermore, although Project Open Source will be a helpful tool after the proposed policy is published, the policy itself needs to address these issues.

According to the Association of University Technology Managers’ Fiscal Year 2014 Licensing Activity Survey, over 900 companies were formed last year as a result of U.S. university licensing activities. A percentage of these startups formed around and were funded for their proprietary source code that resulted, at least in part, from federally funded research. The policy’s requirement that a minimum percentage of each covered agency’s custom code be released as OSS each year may have a notably chilling effect on both the funding and formation of these new companies.

_To what extent is the proposed pilot an effective means to fuel innovation, lower costs, benefit the public, and meet the operational and mission needs of covered agencies?_

The release of custom code as OSS does not guarantee adoption and use by the software community. There are numerous examples of open source code not being used and costing early adaptors significantly more than a proprietary solution alternative. That said, OSS can be extremely beneficial when careful thought is given, on a case-by-case basis, to the nature of the individual custom code to be released.

Additionally, the release of custom code as OSS should have no impact on the adaptation of such code across agencies of the Federal Government. FARS 252.227-7014 already provides for unlimited use by the Federal Government when the noncommercial software is developed solely by government funding. The requirement to release custom code as OSS is duplicative and does not provide a better mechanism for disseminating and adapting software across agencies than that which is already in effect.

_This policy addresses custom code that is created by Federal Government employees as well as custom code that is Federally-procured. To what extent would it be appropriate and desirable for aspects of this draft policy to be applied in the context of Federal grants and cooperative agreements?_

Grants and cooperative agreements are mechanisms used by the Federal Government to provide financial assistance, not to procure products or services for the benefit or use of the Government (31 USC 6304 and 31 USC 6305). We do not believe it would be appropriate to apply this policy to activities not meant for ‘acquiring property or services for the direct benefit or use of the United States Government’.

By the nature of the research performed under Federal grants and cooperative agreements, the application of a policy that mandates release of custom code would have a negative impact on basic research. Decisions will be made based upon expectations that the code may be released as OSS by the Federal Government and not solely on what is best for the direction of the research. Further, while it is not uncommon for source code developed by researchers under grants to be released as OSS, the code is not developed specifically to meet a Government need, often builds upon previous works and may be better disseminated for the public benefit under a different licensing mechanism.

_How can the policy achieve its objectives for code that is developed with Government funds while at the same time enabling the Federal agencies to select suitable software solutions on a case-by-case basis to meet the particular operational and mission needs of the agency?_

As previously discussed, FARS 252.227-7014 already allows for using, sharing and adapting custom code across Federal agencies. This policy will not enhance the Government’s access to code that is already available.

As for mandatory release of custom code, for all of the reasons stated above, we strongly believe that any release as OSS should be voluntary. A voluntary program will allow the institution’s principal investigator (PI) and licensing officer to determine the best means of disseminating any developed code, as well as the appropriate licensing mechanism therefor. The dissemination plan can be discussed if and when Alternative I is requested by the contracting institution.

At the time of procurement, if the Government believes there are suitable software solutions available as OSS, this can be stated in the Request for Proposal and included as a factor in evaluating a proposal. Likewise, if the Government believes that the code to be developed under a particular project would lend itself to being OSS and would fuel innovation and lower the costs of the overall project, that requirement could be a factor in evaluating a proposal.

In conclusion, for the reasons discussed above, we believe the proposed policy may have a negative effect on innovation without necessarily accomplishing the objectives of the policy. We particularly urge flexibility with respect to the minimum percentage approach, and do not believe OSS necessarily is the appropriate default for the release of custom code. It depends on the nature and objectives of the particular custom code and should be made on a case-by-case basis.

Sincerely,
Anthony P. DeCrappeo 
President
Council on Governmental Relations 

Hunter R. Rawlings III
President
Association of American Universities
